keyboards/dtd/ldmlKeyboard.dtd

```diff
@@ -132,7 +132,7 @@ Please see CLDR-15034 for the latest information. -->
 
 <!ELEMENT flick EMPTY >
 <!ATTLIST flick directions NMTOKENS #REQUIRED >
-    <!--@MATCH:any-->
+    <!--@MATCH:regex/(n|e|s|w|ne|nw|se|sw|[ ]+)+-->
```

Comment:
This seems like it would allow `nn` or `nsew` or other arbitrary using the letters `nesw` sequences. I am also unsure on the `@MATCH:regex` comment, whether that is matching the whole string, or a partial string match?

Reply:
it matches on the whole string.

Reply:
```suggestion
    <!--@MATCH:regex/(n|e|s|w|ne|nw|se|sw)([ ]+(n|e|s|w|ne|nw|se|sw))*-->
```

This disallows prefix and suffix spaces but I think that is tidy anyway